src/components/header/createHeader.ts

```diff
@@ -4,9 +4,40 @@ const createHeader = (): HTMLElement => {
   const mainHeader = document.createElement("header");
   mainHeader.classList.add("main-header");
 
-  mainHeader.innerHTML = "<h1>Monuments</h1>";
+  mainHeader.innerHTML = `
+  <h1>Monuments</h1>
+  <ul class='selection-menu'>
+    <li class='selection-menu__anchor-container'>
+      <a class='selection-menu__link' href=''><span>Monuments</span></a>
+    </li>
+    <li class='selection-menu__anchor-container'>
+      <a class='selection-menu__link' href=''><span>Add Monument</span></a>
+    </li>
+  </ul>
+  `;
 
   return mainHeader;
 };
 
+export const createSelectionStatus = (): void => {
+  const header = document.querySelector("h1");
```

Comment:
Un `h1` no es un `header`.

Reply:
¿Por qué mergeáis sin responder a este feedback?

Reply:
Se me ha pasado contestarte, he cambiado el nombre de header, a heading